<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addStudent</class>
 <widget class="QDialog" name="addStudent">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>855</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="NamelineEdit">
   <property name="geometry">
    <rect>
     <x>301</x>
     <y>140</y>
     <width>401</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="IDlineEdit">
   <property name="geometry">
    <rect>
     <x>301</x>
     <y>189</y>
     <width>401</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="PhoneNumberlineEdit">
   <property name="geometry">
    <rect>
     <x>301</x>
     <y>238</y>
     <width>401</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="EmaillineEdit">
   <property name="geometry">
    <rect>
     <x>301</x>
     <y>287</y>
     <width>401</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="GenderlineEdit">
   <property name="geometry">
    <rect>
     <x>301</x>
     <y>336</y>
     <width>401</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Password_YearlineEdit">
   <property name="geometry">
    <rect>
     <x>301</x>
     <y>385</y>
     <width>401</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="GrouplineEdit">
   <property name="geometry">
    <rect>
     <x>301</x>
     <y>434</y>
     <width>401</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Namelabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="IDlabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="Emaillabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>290</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>E-mail</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>240</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Phone Number</string>
   </property>
  </widget>
  <widget class="QLabel" name="Genderlabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>340</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Gender</string>
   </property>
  </widget>
  <widget class="QLabel" name="Password_yearlabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>390</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="GroupLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>430</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Group</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AddButton">
   <property name="geometry">
    <rect>
     <x>642</x>
     <y>560</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
